AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -222,28 +222,18 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
                     rawResult.alternatives.forEach { rawAlternative in
                         var items: [TranscriptItem] = []
                         var entities: [TranscriptEntity] = []
-                        rawAlternative.items.forEach { rawItem in
-                            let item = TranscriptItem(type: Converters.Transcript.toTranscriptItemType(type: rawItem.type),
-                                                      startTimeMs: rawItem.startTimeMs,
-                                                      endTimeMs: rawItem.endTimeMs,
-                                                      attendee: Converters.Transcript.toAttendeeInfo(attendeeInfo: rawItem.attendee),
-                                                      content: rawItem.content,
-                                                      vocabularyFilterMatch: rawItem.vocabularyFilterMatch,
-                                                      stable: rawItem.stable,
-                                                      confidence: rawItem.confidence)
-                            items.append(item)
+                        items = rawAlternative.items.map {
```

Comment:
nit: remove line 223 so that you can have 

`let items = rawAlternative.items.map {...}`

Same for `entities`

Reply:
will update.